torchci/pages/api/drci/drci.ts

```diff
@@ -428,11 +442,28 @@ function constructResultsJobsSections(
   }
 
   output += "<p>\n\n"; // Two newlines are needed for bullts below to be formattec correctly
+
+  const hudPrUrl = `${hudBaseUrl}/pr/${owner}/${repo}/${prNumber}`;
   const jobsSorted = jobs.sort((a, b) => a.name!.localeCompare(b.name!));
   for (const job of jobsSorted) {
-    output += `* [${job.name}](${hud_pr_url}#${job.id}) ([gh](${job.html_url}))\n`;
+    output += `* [${job.name}](${hudPrUrl}#${job.id}) ([gh](${job.html_url}))`;
+
+    const relatedJob = relatedJobs.get(job.id);
+    // Show the related trunk failure for broken trunk or the similar failure for flaky
+    if (relatedJob !== undefined) {
+      const hudCommitUrl = `${hudBaseUrl}/${owner}/${repo}/commit/${relatedJob.head_sha}`;
+      const relatedJobLink = `${hudCommitUrl}#${relatedJob.id}`;
+      if (header === "BROKEN TRUNK") {
+        output += ` ([trunk failure](${relatedJobLink}))`;
+      } else if (header === "FLAKY") {
+        output += `([similar failure](${relatedJobLink}))`;
+      } else {
+        output += `([related job](${relatedJobLink}))`;
+      }
+    }
+
     if (job.failure_captures) {
-      output += `    \`${job.failure_captures[0]}\`\n`;
+      output += `\n    \`${job.failure_captures[0]}\`\n`;
```

Comment:
sanity check, i think you might need a `\n` to divide lines within a category if the failures captures doesnt exist

Reply:
Oh, yeah, good catch.  it would screw up the format otherwise